tests/make_index_test.py

```diff
@@ -80,8 +88,20 @@ def test_main_new_package(tmp_path):
 
     # just some smoke tests about the output
     assert dest.joinpath("packages.json").exists()
-    assert dest.joinpath("wheels/a-1-py3-none-any.whl").exists()
-    assert dest.joinpath("simple/a/index.html").exists()
+    assert dest.joinpath(f"wheels/{wheel_name}").exists()
```

Comment:
I see you DRY'd this up -- imo this makes the test harder to read because you can't see the structure literally